plugins/tutor-contrib-paragon/pyproject.toml

```diff
@@ -28,7 +28,7 @@ classifiers = [
 ]
 dependencies = [
     "tutor>=19.0.0,<21.0.0",
-    "tutor-mfe @ git+https://github.com/overhangio/tutor-mfe.git@release",
+    "tutor-mfe>=20.1.0",
```

Comment:
I think I'm happy with this just being `>=`, as I would expect `tutor-mfe` to continue to support the things we need from it going forward, but I wonder if we should add a `<21.0.0` to match the `tutor` dependency.